app-header/app-header.html

```diff
@@ -95,6 +95,9 @@
       position: relative;
       display: block;
       -webkit-transform: translateZ(0);
+      transition-property: -webkit-transform;
+      transition-property: transform;
```

Comment:
I don't think it'll overwrite if the second one is not recognized (i.e. if the browser does not support the unprefixed `transform`, it'll ignore the second one). Obviously we do want it to override if it is supported.